### ISSUE TITLE
First pass at General Information form in Django app

### ISSUE THREAD

**Issue by tadhg-ohiggins** [GSA-TTS/FAC#563]
Breakout issue of #556

### Background

The current frontend submits data to Django REST Framework endpoints; we think that having the Django version do the same makes sense, but we need to explore that to see if there are hidden obstacles.

This issue covers standing up very basic versions the General Information form in Django. This includes some hack to generate the pre-SAC information in a way that makes it possible to start the General Information entry process.

### Acceptance Criteria (We'll know we're done when...)

- [ ]   We have a working Django application that can accept General Information form data and either send it via the existing DRF endpoints or enter it into the DB some other way, whichever is easier.
- [ ]    No new CSS or JavaScript has been written, as that needs to be covered by the new SASS build pipeline.

### Tasks

- [x]   Figure out the pre-SAC flow we need to work around needing to port the whole existing pre-SAC process over.
- [x]   Stand up a form that can accept the General Information data and get it into the DB.

**Comment by geekygirlsarah** [comment]
@jasnakai What I've learned from chats:
* We should probably put templates/HTMLs in `/backend/report_submission/templates/*`
* Views would also go under `/backend/report_submission/views.py`
* Probably we'd want to take out the API calls and just move them over to Django, with submission via POST to Django, and returning HTML instead of an API call then just GETting to a new page 
* We need to keep CSS/JS, but can delete some JS if needed
* Views will pass over data to the templates to render (as opposed to being filled in via API calls)

I'm happy to show you any of this tomorrow if you want before/after our big 1/2 day meeting.

**Comment by pkigongo** [comment]
Waiting on #638

**Comment by melchoyce** [comment]
Latest mockup:

![image](https://user-images.githubusercontent.com/2846578/214883919-613852c0-54f4-4c09-93bf-0bbaadc419f9.png)
https://www.figma.com/file/1bNncVt1TcFFDhQslqFC2o/FAC-Playground?node-id=4954%3A24912&t=ILmPWtlO7zE5ea1b-4

The side nav here is for within this page, since it's so long, and some of the screens after it change depending on the type of audit you're filing.

Related: #185

**Comment by pkigongo** [comment]
Per Jason "Need to pair with backend dev to get the general info form to submit."

**Comment by JeanMarie-TTS** [comment]
@heymatthenry , i suggest we move outstanding UI issues from this ticket to a separate one so that @timoballard  can focus on wrapping this one up once the backend work is completed. The new ticket can focus on strictly UI issues, and I saw a few on the page.

**Comment by heymatthenry** [comment]
@JeanMarie-TTS No objection here. Let's get this thing across the finish line.
